[2024-02-12T23:00:11.796+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.transform_load_weather_data_to_S3 scheduled__2024-02-12T22:30:00+00:00 [queued]>
[2024-02-12T23:00:11.833+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.transform_load_weather_data_to_S3 scheduled__2024-02-12T22:30:00+00:00 [queued]>
[2024-02-12T23:00:11.835+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2024-02-12T23:00:11.875+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): transform_load_weather_data_to_S3> on 2024-02-12 22:30:00+00:00
[2024-02-12T23:00:11.884+0000] {standard_task_runner.py:60} INFO - Started process 323 to run task
[2024-02-12T23:00:11.891+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'transform_load_weather_data_to_S3', 'scheduled__2024-02-12T22:30:00+00:00', '--job-id', '142', '--raw', '--subdir', 'DAGS_FOLDER/Weather_dag.py', '--cfg-path', '/tmp/tmpr5weemdr']
[2024-02-12T23:00:11.896+0000] {standard_task_runner.py:88} INFO - Job 142: Subtask transform_load_weather_data_to_S3
[2024-02-12T23:00:12.032+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_dag.transform_load_weather_data_to_S3 scheduled__2024-02-12T22:30:00+00:00 [running]> on host 3bd3b2605e24
[2024-02-12T23:00:12.490+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='stellar' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='transform_load_weather_data_to_S3' AIRFLOW_CTX_EXECUTION_DATE='2024-02-12T22:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-12T22:30:00+00:00'
[2024-02-12T23:00:13.461+0000] {python.py:201} INFO - Done. Returned value was: s3://bucket-weather-data/current_weather_data_portland_12022024230013.csv
[2024-02-12T23:00:13.509+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=transform_load_weather_data_to_S3, execution_date=20240212T223000, start_date=20240212T230011, end_date=20240212T230013
[2024-02-12T23:00:13.553+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-12T23:00:13.627+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
